### ISSUE TITLE
Sync-XliffTranslations: xml attribute target-language cannot be found

### ISSUE THREAD

**Issue by tpokorra** [rvanbekkum/ps-xliff-sync#30]
When running Sync-XliffTranslations, for a language where there is no translation file yet, I get this error:

```
>> Sync-XliffTranslations -sourcePath "$GeneralXlf" -targetLanguage "de-DE" -findByXliffGeneratorNoteAndSource -findByXliffGeneratorAndDeveloperNote -findByXliffGeneratorNote
Loading source document C:\...\Translations\My Extension.g.xlf
Creating new document for language 'de-DE'
Ausnahme beim Festlegen von "target-language": "Die Eigenschaft "target-language" wurde für dieses Objekt nicht gefunden. Vergewissern Sie sich, dass die Eigenschaft vorhanden ist und festgelegt werden kann."
In C:\Program Files\WindowsPowerShell\Modules\XliffSync\1.6.0.0\Model\XlfDocument.ps1:707 Zeichen:17
+                 $newFileNode.'target-language' = $language;
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], SetValueInvocationException
    + FullyQualifiedErrorId : ExceptionWhenSetting

Es ist nicht möglich, eine Methode für einen Ausdruck aufzurufen, der den NULL hat.
In C:\Program Files\WindowsPowerShell\Modules\XliffSync\1.6.0.0\Public\Sync-XliffTranslations.ps1:126 Zeichen:5
+     [string] $language = $targetDocument.GetTargetLanguage();
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

Creating Maps in memory for target document's units.
```

Translation of the error message: The propery "target-language" was not found for this object. Please make sure that the property exists and can be set.

Also, if I have generated the translation file in VSCode, and then run the powershell script, the attribute target-language disappears from the translation file.

**Comment by rvanbekkum** [comment]
Thank you for filing the issue and for your PR! 😊

**Comment by rvanbekkum** [comment]
The fix for this bug has been shipped with 1.7.0.0!
